### ISSUE TITLE
jsx-indent behaviour with ternaries

### ISSUE THREAD

**Issue by mklopets** [yannickcr/eslint-plugin-react#907]
Upgrading from v6.3.0 to v6.4.0 made a perfectly sensible way of doing ternary operators in JSX fail linting due to indentation (did not throw error in 6.3.0).

This:

```
this.props.asd.length > 0 ?
    <Button className='bacon-yay'>{this.props.asd.length}</Button> :
    <span className='bacon-no-trigger'>0</span>
```

is supposedly incorrect.

Instead, the linter suggests you should use this, which doesn't make sense:

```
this.props.asd.length > 0 ?
    <Button className='bacon-yay'>{this.props.asd.length}</Button> :
        <span className='bacon-no-trigger'>0</span>
```

**Comment by klimashkin** [comment]
Yep, it's really critical https://github.com/yannickcr/eslint-plugin-react/issues/901
Can't use 6.4.x with it

**Comment by dmt0** [comment]
@yannickcr This issue is anything but fixed. Using 6.10.0, I'm getting this:

    43:7  error  Expected indentation of 4 space characters but found 6  react/jsx-indent

in this code:

              {!condition === 0 ? '' :
                <a href={this.props.exportUrl} >{t('export_btn')}</a>
              }

which I find indented reasonably. Linter expects it to look like this:

              {!condition === 0 ? '' :
              <a href={this.props.exportUrl} >{t('export_btn')}</a>
              }

**Comment by dentuzhik** [comment]
I have the same issue, our code still heavily affected by this, and we can't update from 6.3.0 for the reason that this gives a bunch of errors.

Can we at least reopen that?
I'm willing to contribute with additional info/tests failing, if necessary.

**Comment by dentuzhik** [comment]
Actually NVM, the problem was related to a different issue, seems like `jsx-indent` is actually working fine. Sorry for direct ping.
